2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala

```diff
@@ -117,6 +122,7 @@ package model {
     cookie: CookieConfig,
     cookieBounce: CookieBounceConfig,
     redirectMacro: RedirectMacroConfig,
+    rootResponse: Option[RootResponseConfig],
```

Comment:
we've standardized against enabled-style in #3556, it'd be cool if this new feature could follow this guildeline too :+1:

Reply:
Will do. Ironically, that's what my first pass at this code did :-).